### ISSUE TITLE
🐛 Dark mode color contrast problem on IAP page

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#13592]
### 🐛 Bug and Expected Behaviour

The "go back to English" link on https://talent.canada.ca/en/indigenous-it-apprentice?locale=crk does not have correct contrast in dark mode.

![Image](https://github.com/user-attachments/assets/a22235ec-7e66-4ca8-8e74-fdaa50a907d1)

### ✅ Acceptance Criteria

- [ ] "Go back to English" link stays as White text in dark mode.

**Comment by JoshBeveridge** [comment]
Yep, confirmed with @BrookeCameron that this should just be the same white as the text above in the same block. Likely missing a `:all` modifier on the Hydrogen attribute.

**Comment by petertgiles** [comment]
Note: this may be resolved by the Tailwind Button component conversion

**Comment by esizer** [comment]
> Note: this may be resolved by the Tailwind Button component conversion

Button refactor does fix this.

![Image](https://github.com/user-attachments/assets/2ddff930-45c4-40f0-9a68-9dc5e21f935a)
![Image](https://github.com/user-attachments/assets/306aa87e-a3fd-470c-8e46-c79044b2cd94)
